docs/platform_operators/scaling-networking.md

```diff
@@ -0,0 +1,136 @@
+# Scaling Networking
+
+## Local Development
+If you would like cf-for-k8s to be right-sized for local development on a laptop,
+you can ignore this page and follow the regular deployment docs as by default
+cf-for-k8s is configured for a small footprint deployment. This documentation
+describes how to scale the cf-for-k8s networking components for production use cases.
+
+## Larger Scale Deployments
+If you would like to scale up the networking components for a larger scale
+environment, simply:
+1. Copy the following example into a `scaling-networking.yml` file
+1. Adjust the values to your liking
+1. Append `-f scaling-networking.yml` to the ytt command you run for deployment
+
+The number of ingressgateway replicas depends on load profile of your cluster.
+
+The number istiod replicas depends on the number of application instances.
+
+We recommend two routecontrollers for high availability, but you likely won't
+need more than that.
+
+Sidecar resource usage depends on the load profile of your AIs.
+
+```yaml
+#@ load("@ytt:overlay", "overlay")
+#@ load("@ytt:json", "json")
+
+#! Modify these values to adjust scaling characteristics
+
+#@ ingress_gateway_replicas = 2
+#@ ingress_gateway_cpu_request = "1"
+#@ ingress_gateway_cpu_limit = "2"
+#@ ingress_gateway_mem_request = "1Gi"
+#@ ingress_gateway_mem_limit = "2Gi"
+
+#@ istiod_replicas = 2
+#@ istiod_cpu_request = "1"
+#@ istiod_cpu_limit = "2"
+#@ istiod_mem_request = "1Gi"
+#@ istiod_mem_limit = "2Gi"
+
+#@ routecontroller_replicas = 2
+#@ routecontroller_cpu_request = "200m"
+#@ routecontroller_cpu_limit = "400m"
+#@ routecontroller_mem_request = "32Mi"
+#@ routecontroller_mem_limit = "1024Mi"
+
+#@ sidecar_cpu_request = "100m"
+#@ sidecar_cpu_limit = "2000m"
+#@ sidecar_mem_request = "128Mi"
+#@ sidecar_mem_limit = "1024Mi"
+
+#! DO NOT MODIFY BELOW THIS LINE
```

Comment:
maybe this should say something like "the intent is that the overlays below shouldn't need to be changed"